worker/src/test/java/gov/cms/ab2d/worker/adapter/bluebutton/ContractBeneSearchTest.java

```diff
@@ -119,7 +123,7 @@ void GivenPatientActiveInMonth1And3ButNot2_ShouldReturnTwoRowsInDateRangesUnderC
         Bundle bundle1 = bundle.copy();
         // add 2nd patient
         List<BundleEntryComponent> entries = bundle1.getEntry();
-        entries.add(createBundleEntry("ccw_patient_001", YEAR));
+        entries.add(createBundleEntry("ccw_patient_001", "mbi1", YEAR));
```

Comment:
Refactor to constant both "ccw_patient_001" and "mbi1" as it is used multiple times.